### PR TITLE
Updated WAF feature descriptions and provisioning process to match cu…

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Removed instructions for contacting support to enable the Managed Cloud WAF topic and updated the limitations section to reflect currently functionality. Magento automatically enables the Managed Cloud WAF on all Production environments.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/cloud/cdn/fastly-waf-service.html

### CODE REVIEW

File: guides/v2.2/cloud/cdn/fastly-waf-service.md [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
+-  **WAF onboarding and enablement**—Magento deploys and enables the default WAF policy in your production environment within 2-3 weeks of the environment being fully provisioned.
+-  **Operations and maintenance support**—
+   -  Magento and Fastly set up and manage your WAF logs and alerts.
+   -  Magento triages customer support tickets related to WAF issues that block legitimate traffic as Priority 1 issues.
+   -  Automated WAF version upgrades ensure immediate coverage for new or evolving exploits. See [WAF maintenance and upgrades](#waf-maintenance-and-updates).
 
-Both the {{ site.data.var.ece }} Pro and Starter subscriptions include the Managed Cloud WAF at no additional cost. The WAF service is available only on Production environments.
+The WAF service is available on Production environments for all {{ site.data.var.ece }} Pro and Starter plans at no additional costs.
```
Comment (meker12) [inline guides/v2.2/cloud/cdn/fastly-waf-service.md]:
```suggestion
The WAF service is available on Production environments for all {{ site.data.var.ece }} Pro and Starter plans at no additional cost.
```

File: guides/v2.2/cloud/cdn/fastly-waf-service.md [hunk 2]
```diff
@@ -1,47 +1,45 @@
...
 
 {:.bs-callout .bs-callout-tip}
 For additional information about maintaining PCI compliance for your {{ site.var.data.ece }} stores, see [Magento Approach to PCI compliance](https://magento.com/pci-compliance).
 
 ### Enabling the WAF
 
-Fastly implements the managed web application firewall for {{ site.data.var.ece }} using the Fastly CDN service. You do not have to install or maintain any hardware or software.
+Magento enables the WAF service on new accounts within 2-3 weeks after final provisioning. The WAF is implemented through Fastly CDN service, you do not have to install or maintain any hardware or software.
```
Comment (hguthrie) [inline guides/v2.2/cloud/cdn/fastly-waf-service.md]: When it says "final" provisioning, are there other "provisionings" they will experience? This is worded differently than up above, so makes me think there is a slight difference. It says "within 2-3 weeks of the environment being fully provisioned." I think the one here is more elegant than the earlier one.

```suggestion
Magento enables the WAF service on new accounts within 2 to 3 weeks after provisioning is final. The WAF is implemented through Fastly CDN service; you do not have to install or maintain any hardware or software.
```

File: guides/v2.2/cloud/cdn/fastly-waf-service.md [hunk 3]
```diff
@@ -57,15 +55,15 @@ Magento and Fastly manage the update process to ensure that new or modified WAF
...
-- Protection against malware or bot mitigation
-- Rate limiting or virtual patching
-- Configuring a logging endpoint for customer
+-  Protection against malware or bot mitigation
+-  Rate limiting
+-  Configuring a logging endpoint for customer
 
 Although the WAF does not allow you to block or allow traffic based on IP addresses, you can add access control lists (ACL) and custom VCL snippets to your Fastly service to specify the IP addresses and VCL logic for blocking or allowing traffic. See [Custom Fastly VCL snippets]({{ page.baseurl }}/cloud/cdn/cloud-vcl-custom-snippets.html).
 
+Filtering for TCP, UDP, or ICMP requests is not supported by the WAF. However, this functionality is provided by the built-in DDoS protection included with the Fastly CDN service. See [DDoS protection]({{ page.baseurl }}/cloud/cdn/cloud-fastly.html#ddos-protection).
```
Comment (hguthrie) [inline guides/v2.2/cloud/cdn/fastly-waf-service.md]: Should all instances of WAF be `WAF service` ? If so, there are few sprinkled about that need service after it.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**